README.md

```diff
@@ -133,7 +133,8 @@ As of writing this README, the following methods are supported:
 - `set_transaction_tags` - sets the tags on a transaction
 - `set_budget_amount` - sets a budget's value to the given amount (date allowed, will only apply to month specified by default). A zero amount value will "unset" or "clear" the budget for the given category.
 - `create_manual_account` - creates a new manual account
-- `upload_account_balance_history` - uploads account history csv file for a given account
+- `delete_account` - deletes an account by the provided account id
```

Comment:
Please pull this branch off of your other commits in the other PR, so it doesn't accidentally introduce those changes simultaneously.

Reply:
done!